### PR TITLE
feat: update uv device challenge remediation to CHALLENGE_POLL

### PR DESCRIPTION
**Pull request by magizh-okta**
* updated remediation form name for device challenge poll to CHALLENGE_POLL
to be inline with what the action does.
* totp challenge has challenge-authenticator and push/fastpass has
challenge-poll.

Resolves: OKTA-322766

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-322766](https://oktainc.atlassian.net/browse/OKTA-322766)

### CODE REVIEW

File: src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ export default BaseAuthenticatorView.extend({
 
     const currentAuthenticator = this.options?.appState?.get('currentAuthenticator');
     const selectedMethod = currentAuthenticator?.methods[0];
-    if (selectedMethod?.type === 'totp') {
-      this.Body = ChallengeOktaVerifyTotpView;
+    if (selectedMethod?.type === 'push') {
```
Comment (jhe-okta) [inline src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js:12]: Do you need to update the mock?

File: playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
         "relatesTo": [
           "$.currentAuthenticatorEnrollment"
         ],
-        "name": "challenge-authenticator", 
+        "name": "challenge-poll",
```
Comment (jhe-okta) [inline playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json:16]: Don't you need to update https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/RemediationConstants.js#L21?

Comment (jhe-okta) [reply playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json:16]: I also see that there are some more API mock files still with name `challenge-authenticator`. Can you update them all?

File: src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js (hunk 1 above)
Comment (magizh-okta) [reply src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js:12]: they are updated ( at least the ones relevant to fastpass UV)

File: playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json (hunk 2 above)
Comment (magizh-okta) [reply playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json:16]: @jhe-okta  did I miss any fastpass related mock that I should have updated? This change is not to completely remove 'challenge-authenticator" as it is relevant in a few cases. eg) totp.

All tests are passing and I am not sure what I am missing here.

File: src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js (hunk 1 above)
Comment (jhe-okta) [reply src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js:12]: Can you show me how it is updated? I can't find it from any mock.

File: playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json (hunk 2 above)
Comment (jhe-okta) [reply playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json:16]: I think all the mock files start with `identify-` and contain `challenge-authenticator`.

Comment (magizh-okta) [reply playground/mocks/data/idp/idx/identify-with-user-verification-custom-uri.json:16]: @jhe-okta  I double checked the only files relevant to OV fastpass that has 'challenge-authenticator' have been fixed 

https://github.com/okta/okta-signin-widget/search?l=JSON&p=1&q=challenge-authenticator

**Review by gowthamidommety-okta — APPROVED** [review]
lgtm!

File: src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js (hunk 1 above)
Comment (magizh-okta) [reply src/v2/view-builder/views/ov/ChallengeOktaVerifyView.js:12]: https://github.com/okta/okta-signin-widget/pull/1422/files?file-filters%5B%5D=.json

**Review by jhe-okta — APPROVED** [review]

**Review by santoshmale-okta — APPROVED** [review]

**Pull request closed without merging**